_documentation/HowToUse.md

```diff
@@ -4,336 +4,207 @@ permalink: documentation/HowToUse.html
 title: "Documentation: How To Use"
 ---
 
-#How To Use
+# Table of Contents
 
-##Checking files with the Online GUI 
-<ol type="1">
-  <li>Go here for the latest version: <a href="https://mediaarea.net/MediaConchOnline/checker">https://mediaarea.net/MediaConchOnline/checker</a></li>
-  <li>Register an account by creating  your username / password, which will be emailed to you with instructions. Then log in.</li>
-  <li>Choose a policy</li>
-  <li>Choose a file</li>
-  <ul>
-    <li>You can check a <a href="#check-local-file">local file</a> by selecting the “check file by upload” tab</li>
-    <li>Choose one from <a href="#check-online-file">the Internet</a> by selecting the “check online files” tab, or</li>
-    <li>Use the test files uploaded to the server by selecting the “check server files” tab</li>
-  </ul>
+* [Overview](#overview)
+* [Using MediaConchOnline and MediaConch GUI](#using-mediaconchonline-and-mediaconch-gui)
+* [Using MediaConch CLI](#using-mediaconch-cli)
 
-  <li>Click “Check Files”</li>
-  <li>Once checked, the results will appear below and you can see the reports for each section by clicking on the “eyecon” <img src="../images/eyecon.png" alt="eyecon" /> (3rd screenshot) or download them by clicking the <img src="../images/downArrow.png" alt="down arrow" /></li>
-</ol>
+# Overview
 
-Checker:
+## Installation Guide
 
-![MediaConchOnlineGuiHome](../images/MCOnlineGUI1Home.png)
-![MediaConchOnlineGuiChecked](../images/MCOnlineGUI2Checked.png)
+MediaConch is available for use as a command line interface (CLI), a graphical user interface (GUI) and a web user interface (WebUI). See [our installation guide](https://mediaarea.net/MediaConch/installation.html) how to install with the interface of your choosing.
```

Comment:
2nd sentence seems to be missing a word. Maybe `see our installation guide -on- how to`